dts/bindings/pinctrl/nxp,mci-io-mux.yaml

```diff
@@ -69,3 +69,15 @@ child-binding:
           1 - normal slew rate
           2 - fast slew rate
           3 - fastest slew rate (ultra)
+      sleep-output:
+        type: string
+        default: "low"
+        enum:
+          - "low"
+          - "high"
+          - "unchanged"
+        description: |
+          Sets pin output level in sleep mode.
+          0 - output level low
+          1 - output level high
+          2 - pin configuration is same as in active mode.
```

Comment:
" pin configuration is same as in active mode." not sure if this is the best description, since the pin acts differently than in active mode still, doesn't it?

Reply:
The pin configuration will be left unchanged when we enter sleep mode i.e it will be not be forced to output low.